File: ToDoApp.Avalonia/Models/TodoItem.cs
using System;

namespace ToDoApp.Avalonia.Models;

public class TodoItem
{
    public int Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public string Status { get; set; } = "未着手";
    public string Priority { get; set; } = "中";
    public DateTime? DueDate { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public bool IsCompleted { get; set; }
}

File: ToDoApp.KotlinMultiplatform/composeApp/src/commonMain/kotlin/com/example/todoappkotlinmultiplatform/model/TodoItem.kt
package com.example.todoappkotlinmultiplatform.model

import kotlinx.serialization.Serializable

@Serializable
data class TodoItem(
    val id: Int,
    val title: String,
    val description: String = "",
    val status: TodoStatus,
    val priority: TodoPriority,
    val dueDate: String? = null,
    val createdAt: String,
    val updatedAt: String,
    val isCompleted: Boolean = false
)

@Serializable
enum class TodoStatus(val displayName: String) {
    未着手("未着手"),
    進行中("進行中"),
    完了("完了")
}

@Serializable
enum class TodoPriority(val displayName: String) {
    低("低"),
    中("中"),
    高("高")
}
